From e7662734df72abed3e10dcb032d3fd7284e0e868 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Thu, 11 May 2023 10:03:59 +0200
Subject: parted: link to libuuid

parted.c uses libuuid since 61b3a9733c0e0a79ccc43096642d378c8706add6.
Hence, add UUID_LIBS to PARTED_LIBS to avoid
"DSO missing from command line" error.

Signed-off-by: Brian C. Lane <bcl@redhat.com>

Origin: upstream, https://git.savannah.gnu.org/cgit/parted.git/commit/?id=d22c2d01f62139e0d386d90584cac0705857a571
Last-Update: 2023-06-13

Patch-Name: link-libuuid.patch
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index d838c32c..ea614430 100644
--- a/configure.ac
+++ b/configure.ac
@@ -300,6 +300,8 @@ Note: originally, libuuid was part of the e2fsprogs package.  Later, it
 moved to util-linux-ng-2.16, and that package is now the preferred source.])])
 AC_SUBST([UUID_LIBS])
 
+PARTED_LIBS="$PARTED_LIBS $UUID_LIBS"
+
 dnl Check for libdevmapper
 DM_LIBS=
 if test $ENABLE_DEVICE_MAPPER = yes; then
